doc/source/deployment/setup-deployer.rst

```diff
@@ -68,11 +68,12 @@ Deployer. Set up your workspace with the following steps:
   export SOCOK8S_WORKSPACE_BASEDIR=~/socok8s-workspace
 
 
-Cloning repository
------------------------
+Cloning the repository
+----------------------
 
-To get started, clone the socok8s GitHub repository. This repository uses
```

Comment:
@jsuchome should be able provide you information how to install socok8s thru Zypper install.

Reply:
Really, if this part is not for developers, tell users they can

1. install package from https://download.opensuse.org/repositories/Cloud:/socok8s/openSUSE_Leap_15
or 
2. from the iso image

Reply:
1. available URLs at download.opensuse.org are openSUSE_Leap_15.0 or openSUSE_Leap_15.1. Should users be directed to 15.1?
In the src directory, there are RPMs for Helm, socok8s, and openSUSE-Addon-socok8s, as well as python libraries. Specifically what should a user install?
2. Where is the ISO image?